README.rst

```diff
@@ -411,44 +411,74 @@ Usual workflow with `Collections`_ would be::
 
 Collections are available on project level only.
 
-Frontier
---------
+Frontiers
+---------
 
 Typical workflow with `Frontier`_::
 
-    >>> frontier = project.frontier
+    >>> frontiers = project.frontiers
+
+Get all frontiers from a project to iterate through it::
+
+    >>> frontiers.iter()
+    <list_iterator at 0x103c93630>
+
+List all frontiers::
+
+    >>> frontiers.list()
+    ['test', 'test1', 'test2']
+
+Get a frontier by name::
+
+    >>> frontier = frontiers.get('test')
+    >>> frontier
+    <scrapinghub.client.Frontier at 0x1048ae4a8>
+
+Get an iterator to iterate through a frontier slots::
+
+    >>> frontier.iter_slots()
```

Comment:
I had doubts about it but if we're going to use a separate entity for slots, it looks fine